hack/release-stable-minor.sh

```diff
@@ -34,15 +40,18 @@ fi
 
 if test "$((MINOR % 2))" -eq 0
 then
+	echo "${MAJOR_MINOR} is an EUS release, will create an EUS channel"
 	# 4.even get extended update support: https://access.redhat.com/support/policy/updates/openshift#ocp4_phases
```

Comment:
nit: keep this "what does `MINOR % 2` have todo with EUS?" up at the top of this block, or shift it to the front of the `if` statement or something?